compiler/one-cmds/one-codegen

```diff
@@ -52,20 +52,35 @@ def _get_parser(backends_list):
             backends_name) + ')'
     backend_help_message = 'backend name to use ' + backends_name_message
     parser.add_argument('-b', '--backend', type=str, help=backend_help_message)
+    parser.add_argument(
+        '-T', '--target', type=str, help='run with specific target of the backend')
 
     return parser
 
 
-def _verify_arg(parser, args, cfg_args, backend_args, unknown_args):
+def _verify_arg(parser, args, cfg_args, backend_args, unknown_args, cfg_target_args):
```

Comment:
I think the order looks good. Furthermore, changing the api right away looks fine because it is a private function of this `one-codegen` and it is used at the only one place.